clang/test/CodeGen/AArch64/sve-intrinsics/acle_sve_ld1ro.c

```diff
@@ -15,6 +15,20 @@
 #define SVE_ACLE_FUNC(A1, A2, A3, A4) A1##A2##A3##A4
 #endif
 
+// CHECK-LABEL: @test_svld1ro_mf8(
```

Comment:
Can you add this line to the test:
// RUN: %clang_cc1 -triple aarch64 -target-feature +sme -S -disable-O0-optnone -Werror -Wall -o /dev/null %s
so we make sure we can lower this instruction to assembly

Reply:
Done